osquery/remote/transports/tls.cpp

```diff
@@ -37,7 +37,7 @@ CLI_FLAG(string,
 /// Path to optional TLS server/CA certificate(s), used for pinning.
 CLI_FLAG(string,
          tls_server_certs,
-         "",
+         "/usr/share/osquery/certs/certs.pem",
```

Comment:
Is there an established pattern for defining default flag values that will work on Windows? I guess I could wrap the flag declaration (or just the string here) in a preprocessor guard?